CHANGELOG.md

```diff
@@ -15,27 +15,54 @@ Each version will have a separate `Breaking Changes` section as well. To describ
 ### Breaking changes
 ### Removed
 
+## [0.1.3] - 2024-01-17
+### Added 🎉
+* feat: add rewards-v2 related functionality by @supernovahs in https://github.com/Layr-Labs/eigensdk-rs/pull/221
+  * New methods in `ELChainReader`:
+    * `get_operator_avs_split`
+    * `get_operator_pi_split`
+  * New methods in `ELChainWriter`:
+    * `set_operator_avs_split`
+    * `set_operator_pi_split`
 
+### Changed 🛠
+* feat!: remove delegation manager from `ELChainWriter` by @supernovahs in https://github.com/Layr-Labs/eigensdk-rs/pull/214
+  * `ELChainWriter::new` no longer receives the delegation manager address as first parameter.
+* feat!: change way bindings are generated by @MegaRedHand in https://github.com/Layr-Labs/eigensdk-rs/pull/204
+  * `eigen_utils::core` contains bindings related to core contracts
+  * `eigen_utils::middleware` contains bindings related to middleware contracts
+  * `eigen_utils::sdk` contains bindings related to the SDK (should only be used for testing)
+
+### Documentation 📚
+* docs: add CHANGELOG.md by @lferrigno in https://github.com/Layr-Labs/eigensdk-rs/pull/220
+### Other Changes
+* ci: change docker setup action for official one by @MegaRedHand in https://github.com/Layr-Labs/eigensdk-rs/pull/219
+* docs: add error message for `cargo test` on darwin by @MegaRedHand in https://github.com/Layr-Labs/eigensdk-rs/pull/215
+* test: fix `test_register_and_update_operator` by @ricomateo in https://github.com/Layr-Labs/eigensdk-rs/pull/223
+* chore: update way anvil state dump is generated by @ricomateo in https://github.com/Layr-Labs/eigensdk-rs/pull/222
+* fix: disable doctests on `eigen-utils` by @MegaRedHand in https://github.com/Layr-Labs/eigensdk-rs/pull/227
+* chore: bump version by @MegaRedHand in https://github.com/Layr-Labs/eigensdk-rs/pull/228
+* docs: add GitHub release changelog configuration by @MegaRedHand in https://github.com/Layr-Labs/eigensdk-rs/pull/229
 
 ## [0.1.2] - 2024-11-19
```

Comment:
0.1.2 was released on 10 jan right?

Reply:
Jan 9, I think. It seems we mistakenly used the Go SDK's date on this release. Nice catch!

Reply:
Fixed in https://github.com/Layr-Labs/eigensdk-rs/pull/230/commits/50d0876d65d49d6fe2406f9dc6b481bba9a2e2c7